myriad-scheduler/src/main/java/com/ebay/myriad/configuration/ServiceConfiguration.java

```diff
@@ -129,4 +142,96 @@ public String getServiceOpts() {
     return serviceOptsName;
   }
 
+  /**
+   * Custom Serializer
```

Comment:
Can you please elaborate the javadoc a bit to describe that the class is used for serializing "optional" json properties?